README.md

```diff
@@ -20,7 +20,25 @@ See the documentation for `zicht/cms` (https://github.com/zicht/cms)
 for more information on how the `zicht/page-bundle` relates to the
 other Zicht Bundles
 
+## Voters
+To enable the use of [voters](http://symfony.com/doc/current/security/voters.html) in your configuration you should properly 
+configure `security.yml` or in your config the `security` part
+
+At least you should have these lines
+```
+security:
+    access_decision_manager:
+            strategy: unanimous
+```
+
+## PageVoter
+The PageVoter requires the page to have `Zicht\Bundle\PageBundle\Model\PageInterface` implemented.
+This Voter looks for the isPublic public function to check wether a page can be displayed to the public or not.
```

Comment:
- Format isPublic with code markers: `isPublic`.
- "wether" should be spelled "whether".
- "whether" already implies "or not" so you can leave that out.